src/main/scala/firrtl/Emitter.scala

```diff
@@ -151,6 +151,43 @@ object EmitAllModulesAnnotation extends HasShellOptions {
       helpText = "Run the specified module emitter (one file per module)",
       shortOption = Some("e"),
       helpValueName = Some("<chirrtl|high|middle|low|verilog|mverilog|sverilog>")
+    ),
+    new ShellOption[String](
+      longOption = "emit-module-protobufs",
```

Comment:
```suggestion
      longOption = "emit-modules-protobuf",
```
Slight tweak to make the option match the pattern of `-e` and follow the style of `emit-circuit` and `emit-circuit-protobuf`